src/lib/viewers/BaseViewer.js

```diff
@@ -575,8 +575,8 @@ class BaseViewer extends EventEmitter {
 
         if (this.annotator && this.areNewAnnotationsEnabled()) {
             this.annotator.emit(ANNOTATOR_EVENT.setVisibility, false);
-            this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
```

Comment:
Is this not still needed to exit "create" mode in the annotator?

Reply:
It's already in `disableAnnotationControls`